core/templates/pages/blog-dashboard-page/modal-templates/upload-blog-post-thumbnail.component.spec.ts

```diff
@@ -120,6 +120,25 @@ describe('Upload Blog Post Thumbnail Modal Component', () => {
     fixture.detectChanges();
   });
 
+  it('should remove invalid tags and attributes', () => {
```

Comment:
Given the test name, I think this test is not strong enough, by far. You'd need to check that the resulting svg is as expected. You'd also need to do this test for multiple types of images, each testing a different thing.

Reply:
Added multiple invalid SVG checks in the `svg-sanitizer.spec` file to test the code